targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/device/cmsis.h

```diff
@@ -0,0 +1,12 @@
+/* mbed Microcontroller Library - CMSIS
+ * Copyright (C) 2009-2011 ARM Limited. All rights reserved.
+ *
+ * A generic CMSIS include header, pulling in LPC1768 specifics
+ */
+
+#ifndef MBED_CMSIS_H
+#define MBED_CMSIS_H
+
+#include "MBRZA1LU.h"
```

Comment:
should this include cmsis_nvic.h ? I just found the bug report for RZ A1H, and pointed me this direction (Isent just now the fix for it)

Reply:
I modified this content in commit 1469fcd.